server/src/BGS.Api/appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=localhost;Port=5432;Database=BGS;User Id=postgres;Password=pass123",
```

Comment:
`User Id` - is this space required?
And remove the "ConnectionStrings" section, this setting is not needed in appsettings. Just add a description in the PR about this (we will add this information later for ReadMe) and how to add migration.

Reply:
removed space

Reply:
you have your own appsettings.Personal.json. For prod - env. variables